[FR] Add `Modding/Forum_modding`, `Bubble` (#6328)
ppy/osu-wiki: wiki/Modding/Forum_modding/fr.md

---
tags:
  - mv1
  - modding v1
  - moddingv1
  - forum-based modding
  - old modding system
  - modding basé sur le forum
  - ancien système de modding
---

# Forum modding

> **Forum modding**, également connu sous le nom de *Modding v1*, est un nom familier du système de [modding](/wiki/Modding) et de classement basé sur le forum qui a été principalement utilisé jusqu'au [4 novembre 2017](https://osu.ppy.sh/community/forums/topics/650961?n=7), date à laquelle toutes les [beatmaps](/wiki/Beatmap) nouvelles et inactifs sont passés en [discussions d'une beatmap](/wiki/Beatmap_Discussion) par défaut.

Reviewer: ```suggestion
**Forum modding**, également connu sous le nom de *Modding v1*, est un nom familier du système de [modding](/wiki/Modding) et de classement basé sur le forum qui a été principalement utilisé jusqu'au [4 novembre 2017](https://osu.ppy.sh/community/forums/topics/650961?n=7), date à laquelle toutes les [beatmaps](/wiki/Beatmap) nouvelles et inactifs sont passés en [discussions d'une beatmap](/wiki/Beatmap_Discussion) par défaut.
```
```suggestion
**Forum modding**, également connu sous le nom de *Modding v1*, est un nom familier du système de [modding](/wiki/Modding) et de classement basé sur le forum qui a été principalement utilisé jusqu'au [4 novembre 2017](https://osu.ppy.sh/community/forums/topics/650961?n=7), date à laquelle toutes les [beatmaps](/wiki/Beatmap) nouvelles et inactives sont passées en [discussions d'une beatmap](/wiki/Beatmap_Discussion) par défaut.
```

- Certains moddeurs et membres de la BAT se sont appuyés sur la star priority pour décider de la map à modder ou à examiner ensuite.
- La star priority était une mesure approximative de la popularité d'une beatmap parmi les moddeurs et les mappeurs.

La priorité d'une beatmap peut être augmentée par des mods recevant des kudosu ou des utilisateurs tirant des étoiles de kudosu. Dans les [discussions sur les beatmaps](/wiki/Beatmap_Discussion) modernes, la star priority est comparable au nombre de [hype](/wiki/Beatmap/Hype) d'une beatmap.

### Icônes des fils

La progression d'une beatmap dans la procédure de classement était indiquée par les messages postés par les membres des équipes de classement du beatmap. Chaque message avait une icône personnalisée, qui changeait également l'icône du fil de discussion du forum pour refléter son statut :

- ![](/wiki/shared/icon/star.gif) **[Star](/wiki/Disambiguation/Star)** : la beatmap avait le potentiel pour devenir classé.
- ![](/wiki/shared/icon/bubble.gif) **[Bubble](/wiki/Modding/Bubble)** : la beatmap a été vérifié par un membre de la MAT, BAT ou BN, et pourrait être classé après une autre vérification.

> - ![](/wiki/shared/icon/bubble-pop.gif) **[Bubble pop](/wiki/Modding/Bubble#bubble-pop)** : la beatmap avait des problèmes qui l'empêchaient d'être classé après avoir reçu une bulle.

Reviewer: ```suggestion
- ![](/wiki/shared/icon/bubble-pop.gif) **[Bubble pop](/wiki/Modding/Bubble#bubble-pop)** : la beatmap avait des problèmes qui l'empêchaient d'être classé après avoir reçu une bubble.
```

- Le retour d'information du mappeur pourrait être absent.
- Les mods peuvent être partiellement ou totalement ignorés, intentionnellement ou non.
- Les maps pourraient être classées avec des questions unranked, ou sans l'approbation du ou des nominateurs spécifiques au mode.

Les modifications nécessaires pour résoudre les problèmes les plus courants ont été initiées le [26 avril 2013](https://osu.ppy.sh/community/forums/topics/129625), lorsque [peppy](/wiki/People/peppy) a présenté l'idée du [nouveau système de modding](/wiki/Beatmap_Discussion) (connu ensuite sous le nom de *Modding v2*). Le [21 août 2014](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat), une nouvelle catégorie, [qualifiée](/wiki/Beatmap/Category#qualifiée), a été ajoutée. La catégorie servirait de tampon entre les beatmaps en attente et les beatmaps classés et permettrait des déranks plus fluides. De plus, une nouvelle équipe, la [Quality Assurance Team](/wiki/Modding/Quality_Assurance_Team) (*QAT*), a été créée pour vérifier les beatmaps qualifiés et contrôler le processus de classement.

> Ensuite, le système de discussion des beatmap et le [code de conduite pour le modding et le mapping] (/wiki/Rules/Code_de_conduite_pour_le_modding_et_le_mapping#making-a-mod-post) ont été développés et mis à disposition pour restructurer le modding et en faire une expérience plus agréable pour toutes les personnes impliquées :

Author: ```suggestion
Ensuite, le système de discussion des beatmap et le [code de conduite pour le modding et le mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping#créer-un-mod-post) ont été développés et mis à disposition pour restructurer le modding et en faire une expérience plus agréable pour toutes les personnes impliquées :
```
